deploy/L2TokenBridgeSpell.sol

```diff
@@ -77,6 +82,8 @@ contract L2TokenBridgeSpell {
 
         // sanity checks
         require(address(l2Bridge) == l2Bridge_, "L2TokenBridgeSpell/l2-bridge-mismatch");
+        require(keccak256(bytes(l2Bridge.version())) == keccak256("1"), "L2TokenBridgeSpell/version-does-not-match");
```

Comment:
Since this l2 spell will be deployed, does it make sense to pass the version ("1") as a parameter so this can be reused for later upgrades?

Reply:
It seems a bit weird to be reusing this particular function to init a new implementation given that most of the checks and writes done here would no longer be relevant for it. If we really want on-chain checks for that we could add them to upgradeToAndCall. But that would kind of be breaking our rule of only writing lib functions for the first initialization and may open a pandora box. Deploying a second spell if needed is not necessary that bad given there would be at least a new implementation contract to deploy as well. Or we could just accept that all checks after the initial ones will be done off-chain.

Reply:
ok, resolving.